recipes/biomechanical-analysis-framework/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "biomechanical-analysis-framework" %}
+{% set cxx_name = "libbiomechanical-analysis-framework" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ami-iit/biomechanical-analysis-framework/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: fb12cae8da4146c1bb39f7f71e5e95b43c8e8714a1065fa18dbb307c1e8f4d15
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ cxx_name }}
+    script: build_cxx.sh  # [unix]
+    script: bld_cxx.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage(cxx_name, max_pin='x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ cdt('mesa-libgl-devel') }}  # [linux]
+        - {{ cdt('mesa-dri-drivers') }}  # [linux]
+        - {{ cdt('libselinux') }}  # [linux]
+        - {{ cdt('libxdamage') }}  # [linux]
+        - {{ cdt('libxxf86vm') }}  # [linux]
+        - {{ cdt('libxext') }}     # [linux]
+        - cmake
+        - pkg-config
+        - ninja        
+      host:
+        - yarp
+        - idyntree
+        - spdlog
+        - catch2>=3.3.0
+        - manif
+        - bipedal-locomotion-framework>=0.18.0
```

```suggestion
        - libbipedal-locomotion-framework>=0.18.0
```

See https://github.com/conda-forge/staged-recipes/pull/26377#issuecomment-2115113369 .